paymentsheet/src/test/java/com/stripe/android/paymentsheet/IntentConfirmationHandlerTest.kt

```diff
@@ -58,17 +69,117 @@ class IntentConfirmationHandlerTest {
 
         intentConfirmationHandler.start(
             intent = PaymentIntentFixtures.PI_REQUIRES_PAYMENT_METHOD,
-            paymentSelection = PaymentSelection.Saved(savedPaymentMethod),
+            paymentSelection = PaymentSelection.Saved(
+                paymentMethod = savedPaymentMethod,
+                paymentMethodOptionsParams = paymentMethodOptionsParams,
+            ),
         )
 
         val call = interceptor.calls.awaitItem()
 
         assertThat(call).isEqualTo(
             FakeIntentConfirmationInterceptor.InterceptCall.WithExistingPaymentMethod(
                 initializationMode = initializationMode,
-                shippingValues = shippingDetails.toConfirmPaymentIntentShipping(),
+                shippingValues = ConfirmPaymentIntentParams.Shipping(
+                    name = "John Doe",
+                    address = Address.Builder()
+                        .setLine1("123 Apple Street")
+                        .setLine2("Unit 47")
+                        .setCity("South San Francisco")
+                        .setState("CA")
+                        .setCountry("US")
+                        .setPostalCode("99899")
+                        .build(),
```

Comment:
Decided it would be better to test the result instead of using the function call.